doc/opw_kinematics/opw_kinematics_tutorial.rst

```diff
@@ -0,0 +1,76 @@
+OPW Kinematics Solver for Industrial Manipulators
+==================================================
+
+Intro
+------
+
+Taken from `opw_kinematics <https://github.com/Jmeyer1292/opw_kinematics>`_: 
+A simple, analytical inverse kinematic library for industrial robots with parallel bases and spherical wrists.
+Based on the paper *An Analytical Solution of the Inverse Kinematics Problem of Industrial Serial Manipulators
+with an Ortho-parallel Basis and a Spherical Wrist* by Mathias Brandstötter, Arthur Angerer, and Michael Hofbaur.
+
+Purpose
+-------
+
+This package is meant to provide a simpler alternative to IK-Fast based solutions in situations where one has an
+industrial robot with a parallel base and spherical wrist. This configuration is extremely common in industrial robots.
+
+The kinematics are parameterized by 7 primary values taken directly from the robot's spec sheet and a set of
+joint-zero offsets. Given this structure, no other setup is required.
+
+Installation
+-------------
+
+The `opw_kinematics_plugin https://github.com/JeroenDM/moveit_opw_kinematics_plugin`_ can be installed from packages: ::
+
+  sudo apt install ros-noetic-opw-kinematics-plugin
+
+Usage
+------
+
+- Find the MoveIt `kinematics.yaml <../kinematics_configuration/kinematics_configuration_tutorial.html>`_ file created for your robot.
+- Replace ``kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin`` (or similar) with ``kinematics_solver: moveit_opw_kinematics_plugin/MoveItOPWKinematicsPlugin``
+- Set parameters to describe the geometry of your manipulator. The following is an example for the KUKA KR 6 R700: ::
+
+  manipulator:
+    kinematics_solver: moveit_opw_kinematics_plugin/MoveItOPWKinematicsPlugin
+    opw_kinematics_geometric_parameters:
+      a1:  0.025
+      a2: -0.035
+      b:   0.000
+      c1:  0.400
+      c2:  0.315
+      c3:  0.365
+      c4:  0.080
+    opw_kinematics_joint_offsets: [0.0, -1.57079632679, 0, 0, 0, 0]
+    opw_kinematics_joint_sign_corrections: [-1, 1, 1, -1, 1, -1]
+
+The meaning of the parameters can best be understood with a sketch and some tinkering. The plugin will print ``ROS_ERROR``s on startup if they
```

Comment:
Not sure the `s` works there after the `ROS_ERROR`.

At least GH highlighter seems to get confused.

Reply:
that seems easily confused, but http://rst.ninjs.org/ found some more issues, let's see if that also fixes travis